tests/conftest.py

```diff
@@ -1,11 +1,14 @@
 import hashlib
+from datetime import datetime
 
 import pytest
 from alembic.config import main as alembic
+from dateutil.relativedelta import relativedelta
```

Comment:
Why did you use this library? Seems like standard `datetime` is sufficient for our needs.